docs-utils/update-images.js

```diff
@@ -2,8 +2,9 @@ const fs = require('fs')
 const path = require('path')
 const frontmatter = require('front-matter')
 const imageDownloader = require('image-downloader')
+const { log } = require('console')
 
-const baseURL = 'https://raw.githubusercontent.com/vtexdocs/help-center-content/main'
+const baseURL = 'https://raw.githubusercontent.com/vtexdocs/help-center-content/main/'
 const rootDir = path.resolve(__dirname, '..')
```

Comment:
This variable is not used. It can be removed.

Reply:
Ok, removed.